pkg/apis/v1alpha2/aksnodeclass.go

```diff
@@ -53,7 +53,18 @@ type AKSNodeClassSpec struct {
 	// +optional
 	Kubelet *KubeletConfiguration `json:"kubelet,omitempty" hash:"ignore"`
 	// MaxPods is an override for the maximum number of pods that can run on a worker node instance.
-	// +kubebuilder:validation:Minimum:=0
+	// See minimum + maximum pods per node documentation: https://learn.microsoft.com/en-us/azure/aks/concepts-network-ip-address-planning#maximum-pods-per-node
+	// Defaults if unspecified in the nodeclass are:
+	// _______________________________________________________________
+	// | Network Plugin | Network Plugin Mode     | Default Max Pods |
+	// |----------------|-------------------------|------------------|
+	// | Azure		    | "" aka v1 or NodeSubnet | 30               |
+	// | Azure 		    | "overlay" 			  | 250 			 |
+	// | None			| *						  | 250				 |
+	// | Anything Else  | Anything Else 		  | 110 			 | <--- Being the kubernetes default
+	// |-------------------------------------------------------------|
```

Comment:
![image](https://github.com/user-attachments/assets/aa94e038-b871-4076-85dc-f25139fd50ab)
Looks like this locally, wonder why it looks like this here? Regardless should probably document this in a cleaner way then.

Reply:
Because tabs